app/models/user_auth_token.rb

```diff
@@ -122,13 +122,19 @@ def self.lookup(unhashed_token, opts = nil)
     expire_before = SiteSetting.maximum_session_age.hours.ago
 
     user_token =
-      find_by(
+      where(
         "(auth_token = :token OR
                           prev_auth_token = :token) AND rotated_at > :expire_before",
         token: token,
         expire_before: expire_before,
       )
 
+    if SiteSetting.verbose_auth_token_logging && path = opts.dig(:path)
+      user_token = user_token.annotate("path:#{path}")
```

Comment:
This method can be called in middlewares so we want to add annotation here as well.